src/reports/components/report-sections/collapsible-result-section.tsx

```diff
@@ -4,18 +4,18 @@ import { NamedFC } from 'common/react/named-fc';
 import * as React from 'react';
 
 import { CollapsibleComponentCardsProps } from '../../../DetailsView/components/cards/collapsible-component-cards';
-import { ResultSectionTitle } from '../../../DetailsView/components/cards/result-section-title';
-import { ResultSectionProps } from './result-section';
+import { ResultSectionTitle, ResultSectionTitleProps } from '../../../DetailsView/components/cards/result-section-title';
 import { RulesOnly, RulesOnlyDeps, RulesOnlyProps } from './rules-only';
 
 export type CollapsibleResultSectionDeps = {
     collapsibleControl: (props: CollapsibleComponentCardsProps) => JSX.Element;
 } & RulesOnlyDeps;
 
 export type CollapsibleResultSectionProps = RulesOnlyProps &
-    ResultSectionProps & {
```

Comment:
This import seems to have been incorrect for some time now; needed to fix this result-section no longer exists as of this PR.